src/pages/admin/Stripe/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState, useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { TextField } from '@mui/material';
+
+import StyledButton from 'src/components/shared/StyledButton';
+import { StyleVariants } from 'src/components/shared/StyledButton/types';
+import useToast from 'src/components/shared/toasts/components/ToastProvider/ToastProviderHooks';
+import {
+  useGetApplicationFeeQuery,
+  useUpdateApplicationFeeMutation,
+} from 'src/redux/stripe/stripeService';
+
+import AdminSectionTitle from '../AdminSectionTitle';
+
+import SectionWrapper from './styles';
+
+const minimumFee = 0;
+const maximumFee = 1;
```

Comment:
Since the backend stores the value as (type: 'decimal', scale: 2), input the fee as a percentage (e.g., 5.5% as 5.50).